modules/API/external/external_api.js

```diff
@@ -81,6 +81,26 @@ function changeParticipantNumber(APIInstance, number) {
 }
 
 /**
+ * Generates array with URL params based on the passed config object that will
+ * be used for the Jitsi Meet URL generation.
+ *
+ * @param config {object} the config object.
+ * @returns {Array<string>} the array with URL param strings.
+ */
+function configToURLParamsArray(config) {
+    const params = [];
+    Object.keys(config).forEach( key => {
```

Comment:
for..in will also return strings. It will also return properties in the prototype chain, but how is that a problem? If you create a weird object which inherits from a another's prototype and pass that as config I guess you want all the options, don't you?